src/xrSound/SourceCache.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
```

Comment:
Always put a blank line after `#pragma once`
```suggestion
#pragma once

```